app/models/training_library.rb

```diff
@@ -23,8 +23,9 @@ class TrainingLibrary < ApplicationRecord
   serialize :translations, Hash
 
   validates_uniqueness_of :slug, case_sensitive: false
+  validates_uniqueness_of :name, case_sensitive: false
```

Comment:
why was this added?

Reply:
in order to make sure that two libraries won't contain same name.

![image](https://github.com/WikiEducationFoundation/WikiEduDashboard/assets/119471995/cee3f328-fc8a-4d80-bf2e-398ce2dcafb8)